### PR TITLE
Msgs add technical category

### PR DESCRIPTION
**Pull request by insacuri**
# What and why?

# How to test?

# Trello

### CODE REVIEW

File: response_operations_ui/controllers/message_controllers.py [hunk 1]
```diff
@@ -84,34 +85,38 @@ def get_all_conversation_type_counts(survey_id, conversation_tab, business_id):
         return totals
     except KeyError:
         logger.exception("Response was successful but didn't contain a 'totals' key")
-        raise NoMessagesError
+        raise
```
Comment (creativeappdevamit) [inline response_operations_ui/controllers/message_controllers.py:88]: shouldn't we define what to raise just for the clarity?

**Review by creativeappdevamit — APPROVED** [review]
Looks Ok to me, just one question very minor happy if you want to ignore it.

File: response_operations_ui/controllers/message_controllers.py (hunk 1 above)
Comment (insacuri) [reply response_operations_ui/controllers/message_controllers.py:88]: The stack trace for a key error would say it's missing a 'totals' key so finding out what's wrong should still be very clear.  It's also never once happened as far as I'm aware.

**Pull request merged**